File: python/examples/lists/sort_numbers.py
numbers = [7, 2, -4, 19, 8]
print(numbers)                      # [7, 2, -4, 19, 8]
numbers.sort()
print(numbers)                      # [-4, 2, 7, 8, 19]

numbers.sort(reverse=True)
print(numbers)                      # [19, 9, 7, 2, -4]

numbers.sort(key=abs, reverse=True)
print(numbers)                      # [19, 9, 7, -4, 2]

File: perl-programming/examples/arrays/sort_numbers.pl
#!/usr/bin/perl 
use strict;
use warnings;

my $filename = shift or die "Usage: $0 filename\n";

open(my $fh, "<", $filename)
    or die "Could not open '$filename': $!";

my @numbers;
while (my $line = <$fh>) {
    chomp $line;
    my @num = split / /, $line;
    push @numbers, @num;
}

my @sorted = sort {$a <=> $b} @numbers;

foreach my $n (@sorted) {
    print "$n\n";
}

